kots-exporter/kots-exporter.sh

```diff
@@ -66,7 +66,7 @@ check_postreq(){
     echo ""
     echo "############ CHECKING K8S NAMESPACE and HELM RELEASE ################"
     # check if helm release exists
-    if  [[ "$(helm list -o yaml  | yq '.[].name')" != "$slug" ]]
+    if  [[ "$(helm list -n $namespace -o yaml  | yq '.[].name')" != "$slug" ]]
```

Comment:
When $namespace is added here, we end up[checking for its existence after](https://github.com/CircleCI-Public/server-scripts/pull/39/files#diff-21d41844ae65b67d905f431be47aada36cc81bf1e48ff63ea9a73a62a3726bb2R75) we use it.

I think the error for line 69 is probably still meaningful enough to be useful in the case of namespace being missing.  So we should probably either delete the check at line 75, or make it happen first.

Reply:
@kelvintaywl Kindly move the namespace check before helm release check

Reply:
thank you! i was thinking about this too earlier, but wasnt sure what was best.

yes sure! I will move it then :)